.github/workflows/nightly-win-builds.yml

```diff
@@ -0,0 +1,31 @@
+name: Windows Paraview Nightly
+
+on:
+  schedule:
+    - cron: '0 2 * * *' # Run at 2 am
```

Comment:
(no change needed) Note according to https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows this is UTC (which e.g. for PT is 6 or 7pm).